### DISCUSSION TITLE
Is There a Way to Render Isolated Scenes to Textures?

### DISCUSSION THREAD

**Discussion by zicklag** [bevyengine/bevy#6153] [Q&A] [answered]
For instance, I have a map list, and each map is in it's own world-space, at position (0, 0, 0), and I would like to have each of them in their own isolated bevy world, but take the render of each map from the maps camera to a texture in the UI, to display a view into each map.

Is there a way to create and render isolated worlds?

**Comment by mockersf** [comment, marked as answer]
It would be possible to have isolated world, but it would require a lot of efforts...

What you want could work with [`RenderLayers`](https://docs.rs/bevy/latest/bevy/render/view/struct.RenderLayers.html)

**Reply by zicklag** [reply]
I think render-layers is the closest thing to what I need and will be good enough for most use-cases.

Thanks for the answer!
